source/win-main/tabs-context.ts

```diff
@@ -34,6 +36,27 @@ export default function displayTabsContext (event: MouseEvent, callback: (clicke
       id: 'close-all',
       type: 'normal',
       enabled: true
+    },
+    {
+      type: 'separator'
+    },
+    {
+      label: trans('menu.copy_filename'),
+      id: 'copy-filename',
+      type: 'normal',
+      enabled: true
+    },
+    {
+      label: trans('menu.copy_path'),
```

Comment:
I would suggest to rename that to `menu.copy_abs_path` and give it an according ID, because there might be in the future the need to also enable relative path copying. Other than that, everything perfect!

Reply:
> I would suggest to rename that to menu.copy_abs_path and give it an according ID, because there might be in the future the need to also enable relative path copying.

Will do that! Could you possibly give me a hint on where I can define that ID?

Reply:
I have to manually add those to Zettlr translate -- it's not very automated as of now.